symphony-bdk-spring/symphony-bdk-app-spring-boot-starter/src/main/java/com/symphony/bdk/app/spring/controller/AppAuthController.java

```diff
@@ -0,0 +1,112 @@
+package com.symphony.bdk.app.spring.controller;
+
+import com.symphony.bdk.app.spring.model.AppInfo;
+import com.symphony.bdk.app.spring.model.AppToken;
+import com.symphony.bdk.app.spring.model.JwtInfo;
+import com.symphony.bdk.core.auth.AppAuthSession;
+import com.symphony.bdk.core.auth.ExtensionAppAuthenticator;
+import com.symphony.bdk.core.auth.exception.AuthInitializationException;
+import com.symphony.bdk.core.auth.exception.AuthUnauthorizedException;
+import com.symphony.bdk.core.auth.jwt.UserClaim;
+import com.symphony.bdk.spring.SymphonyBdkCoreProperties;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@Slf4j
+@RestController
+@RequestMapping(value = {"/application", "/applications"})
```

Comment:
I believe that it will be import for developers to quickly identify which endpoints are provided by this starter or not. So we should make it part of the request mapping: `@RequestMapping("/bdk/v1/app/auth")` (as well as versioning)